src/index.html

```diff
@@ -1,13 +1,52 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        class="card__img" 
+        src="images/imac.jpeg" 
+        alt="mac pc"
+      >
+      <p class="card__description">
+        APPLE A1419 iMac 27" Retina 
+        5K Monoblock (MNED2UA/A)
+      </p>
+      <div class="card__code">
+        <p>Product code: 195434</p>
+      </div>
+      <div class="card__rewie">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p>Reviews: 5</p>
+      </div>
```

Comment:
3 points:
- format code properly
- `card__code` — no need to wrap with `p`
- not `card__rewie` — `card__review`
```suggestion
    <div class="card" data-qa="card">
      <img 
        class="card__img" 
        src="images/imac.jpeg" 
        alt="mac pc"
      >

      <p class="card__description">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </p>

      <div class="card__code">
        Product code: 195434
      </div>

      <div class="card__review">
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>

        <p>Reviews: 5</p>
      </div>
```